cmd/apiserver/app/server/run_server_test.go

```diff
@@ -71,11 +71,11 @@ func TestRunServerInstallTPRFails(t *testing.T) {
 		}
 	}
 
-	if getAction != 4 {
-		t.Errorf("Expected 4 'get' action, got %d", getAction)
+	if getAction != 5 {
+		t.Errorf("Expected 5 'get' actions, got %d", getAction)
```

Comment:
Shall we start checking resource types, or at least make sure that all `get`/`create` were for different types?